content/blog/2020-08-27-august-20-community-gems.md

```diff
@@ -0,0 +1,179 @@
+---
+title: August '20 Community Gems
+date: 2020-08-27
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  using CI/CD to validate models, advanced DVC pipeline scenarios, and how CML adds pictures
+  to your GitHub and GitLab comments.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  using CI/CD to validate models, advanced DVC pipeline scenarios, and how CML adds pictures
+  to your GitHub and GitLab comments.
+picture: 2020-08-27/Gems_Aug_20.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/august-20-community-gems/477
+tags:
+  - Discord
+  - Gems
+  - CML
+  - hyperparameters
+  - LFS
+  - pipeline
+---
+
+Here are some of our top Q&A's from around the community. With the launch of
+[CML](https://cml.dev) earlier in the month, we've got some new ground to cover!
+
+## DVC questions
+
+### [Q: What's the relationship between your DVC remote and cache? If I have an external cache, do I really need a DVC remote](https://discordapp.com/channels/485586884165107732/563406153334128681/747588572479094866)
+
+You can think of your DVC remote similar to your Git remote- it's a place to
+backup and share artifacts, like datasets and models, and it gives you methods
+to push and pull those artifacts to and from your team.
+
+By default, `.dvc/cache` is also similar to your `.git` cache. They're both
+_local_ caches that optimize access to files (data and models for DVC cache, and
+code for Git cache).
+
+Your DVC remote is a superset of `.dvc/cache`- everything in your cache is a
+copy of something in your remote (though there may be files in your DVC remote
+that are not in your cache, if you have never attempted to `push` or `pull` them
+locally).
+
+In theory, if you are using an
+[external cache](https://dvc.org/doc/command-reference/destroy#example-external-cache-directory)
+(such as an S3 bucket), and all your projects and all your teammates use that
+external cache, and you _know_ that the storage is highly reliable, you don't
+need to also have a DVC remote. If you have any doubts about access to your
+external cache or its reliability, we'd recommend also keeping a remote.
+
+### [Q: One of my files is an output of a DVC pipeline, and I want to track this file with Git and store it in my Git repository since it isn't very big. How can I make this work?](https://discordapp.com/channels/485586884165107732/563406153334128681/732308317627613235)
+
+If a file (call it `myfile`) is the output of a DVC pipeline, DVC takes several
+steps to remove it from Git tracking. If you remove it from the `.gitignore` and
```

Comment:
would be best to clarify these steps - it would be easier to. explain what needs to be done

If you remove -> if you only remove ...

also, it describes the case for an output that already exists and DVC tracked, I guess we need to make clear

also, in some case some additional steps are needed - `dvc unprotect` for example

Reply:
I've rewritten for clarity and added a bit about `dvc unprotect` (although noting this is a special case). Let me know what you think!